### PR TITLE
Don't fail on bad cleanup

### PR DESCRIPTION
**Pull request by damccorm**
Switch to best effort

### CODE REVIEW

File: Tasks/InstallSSHKeyV0/postinstallsshkey.ts [hunk 1]
```diff
@@ -16,8 +16,9 @@ async function run() {
             sshTool.deleteKey(deleteKey)
         }
     } catch (err) {
-        tl.setResult(tl.TaskResult.Failed, err);
+        // This gets cleaned up by the agent anyways, best effort
```
Comment (zachariahcox) [inline Tasks/InstallSSHKeyV0/postinstallsshkey.ts]: this catches all exceptions -- we only want to ignore the one from ps.kill

**Comment by damccorm** [comment]
Using admin and merging since tests are broken for PR builds in general right now

File: Tasks/InstallSSHKeyV0/postinstallsshkey.ts [hunk 2]
```diff
@@ -9,7 +9,12 @@ async function run() {
         let agentPid: string = tl.getTaskVariable(util.postKillAgentSetting);
         if (agentPid) {
             tl.debug('Killing SSH Agent PID: ' + agentPid);
-            ps.kill(+agentPid);
+            try {
+                ps.kill(+agentPid);
+            } catch (err) {
+                // This gets cleaned up by the agent anyways, best effort
```
Comment (zachariahcox) [inline Tasks/InstallSSHKeyV0/postinstallsshkey.ts]: this would also be a good place to explain the problem -- the ssh pid is specifically unreliable

Comment (zachariahcox) [reply Tasks/InstallSSHKeyV0/postinstallsshkey.ts]: https://www.schoonology.com/technology/ssh-agent-windows/

**Comment by robertoandrade** [comment]
When is this expected to be published? ie: when is the next revision of the Task expected to be tagged and available in Azure DevOps for jobs to consume?

**Comment by damccorm** [comment]
Its already rolled out some places, I'd expect it to be completely rolled out within the next week or 2 at the latest.

**Comment by robertoandrade** [comment]
Even though the pipeline failed to build it?

**Pull request merged**